### PR TITLE
Update Reporting security issues

### PR DESCRIPTION
**Pull request by artursouza**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 1]
```diff
@@ -6,10 +6,68 @@ weight: 3000
 description: "How to report a security concern or vulnerability to the Dapr maintainers."
 ---
 
-The Dapr organization and team makes security a central focus of how we operate and design our software. From the Dapr binaries to the GitHub release processes, we take numerous steps to ensure user applications and data is secure. For more information visit the [security page]({{< ref security-concept.md >}}).
+The Dapr project and maintainers make security a central focus of how we operate and design our software. From the Dapr binaries to the GitHub release processes, we take numerous steps to ensure user applications and data is secure. For more information on how security is built-in to Dapr, visit the [security page]({{< ref security-concept.md >}}).
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
The Dapr project and maintainers make security a central focus of how we operate and design our software. From the Dapr binaries to the GitHub release processes, we take numerous steps to ensure user applications and data is secure. For more information on Dapr security features, visit the [security page]({{< ref security-concept.md >}}).
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 2]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
 
-## Reporting security issues
+## Covered Repositories and Issues
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
## Repositories and issues covered
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 3]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
 
-To report a security issue, please privately email the [Dapr Maintainers (dapr@dapr.io)](mailto:dapr@dapr.io?subject=[Security%20Disclosure]:%20ISSUE%20TITLE)
+When we say "a security vulnerability in Dapr" we mean a security issue
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
When we say "a security vulnerability in Dapr", this means a security issue
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 4]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+in any repository under the [dapr GitHub organization](https://github.com/dapr/).
 
-The Dapr maintainers will triage and respond ASAP and then patch and send an announcement within 30 days.
+This reporting process is intended only for security issues in the Dapr
+project itself, and doesn't apply to applications _using_ Dapr or to
+issues which do not affect security.
+
+If the issue cannot be fixed by a change to one of the covered
+repositories above, then it recommended to create a GitHub issue in the appropriate repo or a question in Discord.
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
repositories above, then it recommended to create a GitHub issue in the appropriate repo or ask a question on Discord server.
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 5]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+caution!
+
+### Explicitly Not Covered: Vulnerability Scanner Reports
+
+We do not accept reports which amount to copy and pasted output from a vulnerability
+scanning tool **unless** work has specifically been done to confirm that a vulnerability
+reported by the tool _actually exists_ in Dapr, including CLI, SDKs, components-contrib
+or any other repo under the Dapr org.
+
+We make use of these tools ourselves and try to act on the output they produce; they
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
We make use of these tools ourselves and try to act on the output they produce.
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 6]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+If we receive a report which seems to simply be a vulnerability list from a scanner we
+reserve the right to ignore it.
+
+This applies especially when tools produce vulnerability identifiers which are not publicly
+visible or which are proprietary in some way. We can look up CVEs or other publicly-available
+identifiers for further details, but cannot do the same for proprietary identifiers.
+
+## Security Contacts
+
+The people who should have access to read your security report are listed in [`maintainers.csv`](/maintainers.csv).
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
The people who should have access to read your security report are listed in [`maintainers.md`](https://github.com/dapr/community/blob/master/MAINTAINERS.md).
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 7]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+
+## Reporting Process
+
+1. Describe the issue in English, ideally with some example configuration or
+   code which allows the issue to be reproduced. Explain why you believe this
+   to be a security issue in Dapr, if that's not obvious.
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
   to be a security issue in Dapr.
```

File: daprdocs/content/en/operations/support/support-security-issues.md (hunk 4 above)
Comment (hhunter-ms) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
repositories above, then it's recommended to create a GitHub issue in the appropriate repo or raise a question in Discord.
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 8]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
 
-The Dapr maintainers will triage and respond ASAP and then patch and send an announcement within 30 days.
+This reporting process is intended only for security issues in the Dapr
+project itself, and doesn't apply to applications _using_ Dapr or to
+issues which do not affect security.
+
+If the issue cannot be fixed by a change to one of the covered
+repositories above, then it recommended to create a GitHub issue in the appropriate repo or a question in Discord.
+
+All that said, **if you're unsure** please reach out using this process before
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
**If you're unsure,** err on the side of caution and reach out using the reporting process before
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 9]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+raising your issue through another channel. We'd rather err on the side of
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
raising your issue through GitHub, Discord, or another channel. 
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 10]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+caution!
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 11]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+
+### Explicitly Not Covered: Vulnerability Scanner Reports
+
+We do not accept reports which amount to copy and pasted output from a vulnerability
+scanning tool **unless** work has specifically been done to confirm that a vulnerability
+reported by the tool _actually exists_ in Dapr, including CLI, SDKs, components-contrib
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
reported by the tool _actually exists_ in Dapr, including the CLI, Dapr SDKs, the components-contrib repo,
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 12]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+or any other repo under the Dapr org.
+
+We make use of these tools ourselves and try to act on the output they produce; they
+can be useful! We tend to find, however, that when these reports are sent to our security
+mailing list they almost always represent false positives, since these tools tend to check
+for the presence of a library without considering how the library is used in context.
+
+If we receive a report which seems to simply be a vulnerability list from a scanner we
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
If we receive a report which seems to simply be a vulnerability list from a scanner, we
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 13]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+1. Describe the issue in English, ideally with some example configuration or
+   code which allows the issue to be reproduced. Explain why you believe this
+   to be a security issue in Dapr, if that's not obvious.
+2. Put that information into an email. Use a descriptive title.
+3. Send the email to [Dapr Maintainers (dapr@dapr.io)](mailto:dapr@dapr.io?subject=[Security%20Disclosure]:%20ISSUE%20TITLE)
+
+## Response
+
+Response times could be affected by weekends, holidays, breaks or time zone
+differences. That said, the maintainers team will endeavour to reply as
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
differences. That said, the maintainers team endeavours to reply as
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 14]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+soon as possible, ideally within 3 working days.
+
+If the team concludes that the reported issue is indeed a security
+vulnerability in a Dapr project, at least two members of the maintainers
+team will discuss the next steps together as soon as possible, ideally
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
team discuss the next steps together as soon as possible, ideally
```

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 15]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+within 24 hours.
+
+As soon as the team decides that the report is of a genuine vulnerability,
+one of the team will respond to the reporter acknowledging the issue and
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
one of the team responds to the reporter acknowledging the issue and
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
nits

File: daprdocs/content/en/operations/support/support-security-issues.md [hunk 16]
```diff
@@ -6,10 +6,68 @@ weight: 3000
...
+
+### Explicitly Not Covered: Vulnerability Scanner Reports
+
+We do not accept reports which amount to copy and pasted output from a vulnerability
+scanning tool **unless** work has specifically been done to confirm that a vulnerability
+reported by the tool _actually exists_ in Dapr, including the CLI, Dapr SDKs, the components-contrib repo,
+or any other repo under the Dapr org.
+
+We make use of these tools ourselves and try to act on the output they produce; they
+can be useful! We tend to find, however, that when these reports are sent to our security
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-security-issues.md]:
```suggestion
We tend to find, however, that when these reports are sent to our security
```

**Review by msfussell — APPROVED** [review]
LGTM

**Review by hhunter-ms — APPROVED** [review]

**Comment by marcduiker** [comment]
@holopin-bot @artursouza Thanks Artur! :)

**Pull request merged**